.github/workflows/core-api.build.deploy.dev.yaml

```diff
@@ -77,8 +82,8 @@ jobs:
 
   run-if-failed:
     runs-on: ubuntu-20.04
-    needs: [trigger-gitops]
-    if: always() && (needs.trigger-gitops.result == 'failure')
+    needs: [build-backend, build-flyway, trigger-gitops]
```

Comment:
specifically _didn't_ put schemaspy-rollout in here because this notification will say that _core-api_ didn't deploy, and schemaspy has its own notification anyway